paddle/fluid/pybind/eager.cc

```diff
@@ -155,52 +187,175 @@ void InitEagerTensorWithEagerTensor(EagerTensorObject* self,
   }
 }
 
-void InitEagerTensorWithFrameworkTensor(EagerTensorObject* self,
-                                        const framework::Tensor& src,
-                                        const paddle::platform::Place& place,
-                                        const std::string& name) {
-  self->eager_tensor.set_name(name);
-  if (place == src.place()) {
-    std::shared_ptr<pten::DenseTensor> dense_tensor =
-        std::make_shared<pten::DenseTensor>(
-            pten::make_intrusive<paddle::experimental::SharedStorage>(place),
-            pten::DenseTensorMeta(pten::TransToPtenDataType(src.type()),
-                                  src.dims()));
-    paddle::experimental::ReMakePtenDenseTensor(src, dense_tensor.get());
-    self->eager_tensor.set_impl(dense_tensor);
-    VLOG(4) << "Same place, do ShareDataWith";
+// initialize EagerTensor by PyArray (mix args and kwargs) automatically
+void AutoInitEagerTensorByPyArray(
+    EagerTensorObject* py_tensor_ptr,
+    std::unordered_map<std::string, PyObject*> kws_map, PyObject* args,
+    bool flag_kwargs, Py_ssize_t args_num) {
+  // key_word lists: [value, place, persistable, zero_copy, name, stop_gradient]
+  std::unordered_map<std::string, Py_ssize_t> kw_order_map{
```

Comment:
Explain your design in comment, what do u mean with these key-value

Reply:
key is every arguments of the constructor,
 value is the position of the arguments respectively.
If args_num is 3, means that there are 3 position arguments, so we can parse args from "args" by position.